### PR TITLE
Circleci pipeline

### PR DESCRIPTION
**Pull request by mmpetarpeshev**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ COPY --from=frontend /app/assets ./assets
 # copy generated index page
 COPY --from=frontend /app/templates ./templates
 # copy package.json & package-lock.json
-COPY package*.json .
+COPY package*.json ./
```
Comment (venimus) [inline Dockerfile:20]: weird

**Review by dincho — COMMENTED** [review]
LGTM

Please set the correct branch in the workflow

File: Dockerfile (hunk 1 above)
Comment (mmpetarpeshev) [reply Dockerfile:20]: Not sure why we need to copy more than one packages json file , it was in the original Dockerfile , but  when are copied multiple files with COPY command , the path needs end with "/" at end to work.

**Comment by mmpetarpeshev** [comment]
> LGTM
> 
> Please set the correct branch in the workflow

Sure , will changed it.

**Review by venimus — APPROVED** [review]

**Pull request merged**